### PR TITLE
Updating Prow docs with new job config

### PR DESCRIPTION
**Pull request by aireilly**
Updating Prow docs with new job config

### CODE REVIEW

File: scripts/prow/README.adoc [hunk 1]
```diff
@@ -47,6 +47,29 @@ For more information, see link:https://steps.ci.openshift.org/job?org=openshift&
 $HOME/release/ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-enterprise-4.15.yaml
 ----
 
+. Log in to the OpenShift CI cluster. Run the following command:
```
Comment (kalexand-rh) [inline scripts/prow/README.adoc:50]:
```suggestion
. Navigate to the OpenShift CI cluster at this URL and log in by using SSO:
```

File: scripts/prow/README.adoc [hunk 2]
```diff
@@ -47,6 +47,29 @@ For more information, see link:https://steps.ci.openshift.org/job?org=openshift&
...
++
+[source,terminal]
+----
+https://oauth-openshift.apps.ci.l2s4.p1.openshiftapps.com/oauth/token/request
+----
+
+. Click `Display Token` and copy the text of the `oc login` command.
```
Comment (kalexand-rh) [inline scripts/prow/README.adoc:57]: So, back around line 10, add installing `oc` as a prereq.

**Comment by aireilly** [comment]
This update is no longer required :roll_eyes: 

See resolution in https://redhat-internal.slack.com/archives/CFUGK0K9R/p1719820136662409

FOR YOUR INFORMATION @Srivaralakshmi please update your local procedure to remove these steps.

**Comment by aireilly** [comment]
Reopening for other Prow doc updates

**Pull request merged**